docs/sdk/src/reference_docs/development_environment_advice.rs

```diff
@@ -39,6 +39,8 @@
 //!   // See the polkadot-sdk CI job that checks formatting for the current version used in
 //!   // polkadot-sdk.
 //!   "rust-analyzer.rustfmt.extraArgs": ["+nightly-2024-01-22"],
+//!   // useful when writing documentation
+//!   "rust-analyzer.semanticHighlighting.doc.comment.inject.enable": true,
```

Comment:
Could you double check this does something? Should be `true` by default on latest RA version according to https://rust-analyzer.github.io/manual.html

Reply:
Yeah you are right, thanks for pointing it out.